Build/Projects/FrameworkReferences.Net.definition

```diff
@@ -7,12 +7,14 @@
     <Reference Include="System.Core" />
     <Reference Include="Xamarin.iOS" />
     <Reference Include="Lidgren.Network" />
+    <Reference Include="MonoGame.Framework.Core" />
```

Comment:
You don't need to do this here, you can just add `<Reference Include="MonoGame.Framework.Core" />` outside all of the platform-specific blocks.